app/models/item.rb

```diff
@@ -1,5 +1,11 @@
+# app/models/item
+
 class Item < ApplicationRecord
   has_many :invoice_items, dependent: :destroy
   has_many :invoices, through: :invoice_items
   belongs_to :merchant
+
+  def self.not_yet_shipped
+    joins(:invoice_items, :invoices).where("invoice_items.status = 'packaged'").merge(Invoice.order(:created_at))
```

Comment:
Could you add a comment clarifying what this is intended to output? I could help out to refine it from there